it/v2/connectors/PersistenceConnectorSpec.scala

```diff
@@ -81,22 +81,22 @@ class PersistenceConnectorSpec
     with ScalaCheckDrivenPropertyChecks
     with CommonGenerators {
 
-  lazy val appConfig: AppConfig = app.injector.instanceOf[AppConfig]
-
+  lazy val appConfig: AppConfig                           = app.injector.instanceOf[AppConfig]
+  lazy val messageType                                    = MessageType.DeclarationAmendment
   lazy val persistenceConnector: PersistenceConnectorImpl = new PersistenceConnectorImpl(httpClientV2, appConfig, new TestMetrics())
   implicit lazy val ec: ExecutionContext                  = app.materializer.executionContext
 
   "POST /traders/:eori/movements/departures" - {
 
-    lazy val okResultGen =
+    lazy val okResult =
```

Comment:
More a comment, but I'm unsure as to why you renamed this. The reason it was `okResultGen` is because it returns a `Gen[DeclarationResponse]`, and that `okResult` is used in the lambda in line 99, which then shadows this declaration.

It's not a huge problem as this variable shouldn't be used within the test itself, but in general I'd recommend not shadowing variables, it just gets confusing if you do.

Reply:
reverted back to original